tests/Unit/CreateUserTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\Artisan;
+use Tests\TestCase;
+use App\Services\UserService;
+
+
+class CreateUserTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private $userService;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->userService = $this->app->make(UserService::class);
+        Artisan::call('passport:install');
+
+    }
+
+    public function testExample()
+    {
+        $data = [
+            'email' => 'qwerty1234@gmail.com',
+            'password' => 'qwerty1234',
+        ];
+        $user = $this->userService->createUser($data);
+        $this->assertInstanceOf(User::class, $user);
+    }
```

Comment:
You are missing assertDatabaseHas test in here to see if you have created the user in your db